client/components/CompanyComments/CompanyComments.tsx

```diff
@@ -1,11 +1,16 @@
 import React, { Component, FormEvent } from 'react';
+import styled from 'styled-components';
 import {
   withAC,
   ApplicationContainerProps,
 } from '../../containers/ApplicationContainer';
 import Comment from './Comment';
 import CommentForm from './CommentForm';
 
+const CommentSectionWrapper = styled.div`
+  margin-top: 15%;
```

Comment:
Any reason this is a %? This may get unpredictable. @albertzhidong

Reply:
nah we can switch to 15vh -- will do

Reply:
done

Reply:
Mm still not a fan--can we move to ems? It doesn't really make sense to have the padding be a function of vertical space on screen since we scroll vertically imo